source/extensions/filters/http/adaptive_concurrency/concurrency_controller/gradient_controller.cc

```diff
@@ -37,13 +37,12 @@ GradientControllerConfig::GradientControllerConfig(
           proto_config.concurrency_limit_params(), max_concurrency_limit, 1000)),
       min_rtt_aggregate_request_count_(
           PROTOBUF_GET_WRAPPED_OR_DEFAULT(proto_config.min_rtt_calc_params(), request_count, 50)),
-      max_gradient_(PROTOBUF_GET_WRAPPED_OR_DEFAULT(proto_config.concurrency_limit_params(),
-                                                    max_gradient, 2.0)),
       sample_aggregate_percentile_(
           PROTOBUF_PERCENT_TO_DOUBLE_OR_DEFAULT(proto_config, sample_aggregate_percentile, 50)),
-      min_concurrency_(PROTOBUF_GET_WRAPPED_OR_DEFAULT(proto_config.min_rtt_calc_params(),
-                                                       min_concurrency, 3)) {}
-
+      min_concurrency_(
+          PROTOBUF_GET_WRAPPED_OR_DEFAULT(proto_config.min_rtt_calc_params(), min_concurrency, 3)),
+      min_rtt_buffer_pct_(
+          PROTOBUF_PERCENT_TO_DOUBLE_OR_DEFAULT(proto_config.min_rtt_calc_params(), buffer, 25)) {}
```

Comment:
In the docs you say the default is 10. Should it be 10 here or 25 in the docs?

Reply:
Ah, oversight on my part. 25% everywhere.